TheraLang.Tests/DataBuilders/ResourcesBuilders/UserTestBuilder.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using TheraLang.DAL.Entities;
+
+namespace TheraLang.Tests.DataBuilders.ResourcesBuilders
+{
+    public class UserTestBuilder : IDataBuilder<User>
+    {
+        private User _user;
+        public UserTestBuilder()
+        {
+            _user = new User();
+        }
+
+        public UserTestBuilder WithDefault()
+        {
+            _user.PasswordHash = "123456789";
+            _user.Email = "zwichainyi@utmm.com";
+
+            return this;
+        }
+
+        /// <summary>
+        /// Creats deafultUserDetails only was created UserId.
```

Comment:
Okay, it's just a comment, just a comment

Reply:
Srry for my accent, homie